tests/rptest/services/admin.py

```diff
@@ -174,7 +174,7 @@ def _get_stable_configuration(
                 self.redpanda.logger.debug(f"doesn't have leader")
                 return None
             if last_leader < 0:
-                last_leader = meta["leader_id"]
+                last_leader = int(meta["leader_id"])
```

Comment:
`if meta["leader_id"] < 0` already exists above suggesting the type is numeric. why is conversion necessary?